test/controllers/graphql_controller_2_test.rb

```diff
@@ -1157,7 +1157,7 @@ def setup
     team = create_team
     create_team_user team: team, user: u, role: 'owner'
     p = create_project team: team
-    create_project_media archived: true, project: p
+    create_project_media archived: 1, project: p
```

Comment:
Why not use the constants in the test too?

Reply:
Fixed by using constants